### ISSUE TITLE
Error "No storage information found for this user address." when accessing my HTTP RS server from HTTPS app

### ISSUE THREAD

**Issue by stokito** [remotestorage/remotestorage.js#1284]
I installed my own RS server and tried it LiteWrite with an address like stokito@192.168.1.1:8000
and got the error. But in fact it's just a Mixed Content error:

>  was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://192.168.1.1:8000/.well-known/host-meta?resource=acct:%20stokito@192.168.1.1:8000'. This request has been blocked; the content must be served over HTTPS.

Is it possible to make the error message shown to a user more clear? Because the "storage info" is exists but it was a configuration problem that causes the browser failure.

**Comment by DougReeder** [comment]
I believe this is the same issue as https://github.com/remotestorage/armadietto/issues/94

**Comment by raucao** [comment]
Good idea to make that more explicit. Can even sound more technical, because the people seeing it would usually be self-hosters at least, if not developers mostly.
